1-ECommerce/src/App.tsx

```diff
@@ -0,0 +1,14 @@
+import "./index.css";
+import { Filters } from "./components/Filters"
+
+export default function App() {
+    return (
+        <>
+            <div className="contaiener py-8 w-full h-full lg:flex lg:flex-col">
+                <div className="lg:mx-auto">
+                    <Filters />
+                </div>
+            </div>
+        </>
```

Comment:
No need for the Fragment here, we can remove the `<>`

Reply:
Also, could we merge these 2 divs into one? Usually when we have a `div` that only has another `div` inside, the one inside it is redundant.